### ISSUE TITLE
add command should only be available to people with `StressedOut` role (and admins of server)

### ISSUE THREAD

**Issue by armaanbadhan** [armaanbadhan/Stress-Me-Out#23]
# Current Behavior

everyone can use the command and add anything to the database

**Comment by GauravGupta993** [comment]
please assign this issue to me.

**Comment by armaanbadhan** [comment]
good luck! make sure to create pr before 15th midnight

**Comment by GauravGupta993** [comment]
In the code written in the commit made, there is an error that does not recognize StressedOut as a valid permission name. After removing it from the code it works perfectly fine i.e. only allowing administrator to use add command. Also when I created a dummy discord server for testing and added this bot then the role 'StressedOut' is not automatically being created.

**Comment by armaanbadhan** [comment]
yeah, the role isnt created when the bot joins a server in offline state

**Comment by GauravGupta993** [comment]
> yeah, the role isnt created when the bot joins a server in offline state

Really sorry for the trouble but unable to understand what offline state means in context because the computer was connected to the internet while running the code or offline state means some lack of permission granted to the bot during setting up.

**Comment by armaanbadhan** [comment]
i meant when the bot is offline, you dont need to worry about that, you can manually create the role for testing rn
